.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.scm]
+indent_style = space
+indent_size = 4
```

Comment:
I had messed up the indentation in a previous PR. Hopefully this will help to prevent that from happening again.